pkg/email/templates/invitation.html.tmpl

```diff
@@ -0,0 +1,33 @@
+{{- define "title" -}}
+Invitation
+{{- end -}}
+
+{{- define "preview" -}}
+You have been invited {{- with .SenderIds }} by {{ .IDString }} {{ end -}} to join {{ .Network.Name }}.
+{{- end -}}
+
+{{- define "body" -}}
+<p>
+Hello,
+</p>
+<p>
+You have been invited {{- with .SenderIds }} by {{ .IDString }} {{ end -}} to join {{ .Network.Name }}.
```

Comment:
```suggestion
You have been invited {{- with .SenderIds }} by <code>{{ .IDString }}</code> {{ end -}} to join <b>{{ .Network.Name }}</b>.
```
Ideally, we would use the full user name here if available and the id only as fallback.

Reply:
We only have the IDs here.